src/components/MenuContent/Menu.style.tsx

```diff
@@ -103,10 +103,21 @@ export const ParagraphCopy = styled.p`
   margin-bottom: 2rem;
   text-align: ${props => props.theme.megaMenu.paragraphAlignment};
 
-  p {
+  p,
+  a {
     color: ${props => props.theme.megaMenu.secondaryText};
   }
 
+  a {
+    text-decoration: underline;
+  }
+
+  &:hover {
+    a {
+      color: ${props => props.theme.megaMenu.secondaryText};
+    }
+  }
```

Comment:
optional nit: Consider consolidating this with the styles for the `a` element above

```
a {
  text-decoration: underline;
  &:hover {
    color: ${props => props.theme.megaMenu.secondaryText};
  }
}
```